src/System.Management.Automation/engine/InitialSessionState.cs

```diff
@@ -3937,7 +3937,8 @@ internal void UpdateTypes(ExecutionContext context, bool updateOnly)
 
             if (errors.Count > 0)
             {
-                var allErrors = new StringBuilder('\n');
+                // Note: don't use '\n' ('char' convert implicitly to 'ushort' and 'int' and initialize 'StringBuilder' size)
```

Comment:
I think removing the comment is sufficient as it's mostly obvious that a string argument initializes the instance.

If you'd like to make other improvements though - it's good practice to pass the estimated capacity when constructing a `StringBuilder` to minimize extra allocations, but only if it's easy to estimate (e.g. use a constant, avoid computing the "estimate").

And calling `Append` with a char might be slightly better than passing a string to the constructor, but honestly the performance here doesn't really matter.